.pmd/exclude.properties

```diff
@@ -0,0 +1,483 @@
+org.apache.maven.AbstractMavenLifecycleParticipant=AbstractClassWithoutAbstractMethod
```

Comment:
I am not sure that all of those (or even the majority) are actually in the code base.

If you want to add supressions, add `@SuppressWarnings("PMD.<xxx>") statements at the places where the problems occur. That way, an IDE can pick them up and point out when a suppression is no longer needed.

Having a big central file is a sure recipe for this to fall into disrepair and things never getting fixed. Which is counter to what you are trying to accomplish. 

Suppressions need to be applied surgically in the right place where a violation exist. Not as a blanked "We cover the whole class"

Reply:
> place where a violation exist

is only possible doing it here. Other wise we have 500 operations inside the code thats impossible.